graylog2-server/src/main/java/org/graylog2/indexer/results/ScrollResult.java

```diff
@@ -41,12 +41,13 @@
     };
 
     public interface Factory {
-        ScrollResult create(SearchResult initialResult, String query, List<String> fields);
+        ScrollResult create(SearchResult initialResult, @Assisted("query") String query, @Assisted("scroll") String scroll, List<String> fields);
```

Comment:
I will check if that's possible. If not, we still might want to do it. The factory has been introduced in 2.3 and is probably not widely used by plugin authors.